scripts/bind/pic/template.hbs

```diff
@@ -39,7 +42,13 @@ impl PicCanisterTrait for {{PascalCase canister_name}}Pic {
 impl {{PascalCase canister_name}}Pic {
   {{#each methods}}
   pub fn {{this.name}}(&self, _caller: Principal{{#each this.args}}, {{this.0}}: &{{this.1}}{{/each}}) -> Result<{{vec_to_arity this.rets}}, String> {
-      self.update(self.canister_id, "{{escape_debug this.original_name}}", arg0)
+      {{#if this.args.[1]}}
+      unimplemented!("The update function handles only a single argument.")
+      {{else if this.args.[0]}}
+      self.update(self.canister_id, "{{escape_debug this.original_name}}", {{#each this.args}}{{this.0}}{{/each}})
+      {{else}}
+      self.update(self.canister_id, "{{escape_debug this.original_name}}", ())
+      {{/if}}
```

Comment:
The template on the left supports only a single argument called `arg0`.  On the right, we support zero arguments, one argument with any name, but not (yet) more arguments.  Supporting more arguments will be needed, but requires updating the `update()` function, and I don't want to make this PR too big.